connection.js

```diff
@@ -9,7 +9,7 @@ const table_name = "countries"
 const insert_default_data = async () => {
   const { data, error } = await supabase
     .from(table_name)
-    .upsert(
+    .insert(
```

Comment:
In what cases should we decide use between `insert` and `upsert`?

Reply:
upsert is for update, insert is when we are inserting data for the first time.